python.spec

```diff
@@ -74,7 +74,7 @@ sed -ibak "s/ndbm_libs = \[\]/ndbm_libs = ['gdbm', 'gdbm_compat']/" setup.py
 # macros on Linux. The following problem does not exists on BSD machines as
 # cdefs.h does not define these macros.
 case %cmsplatf in
-  slc6*|fc*)
+  slc7*|slc6*|fc*)
```

Comment:
Again, I would make an empty `osx*` stanza and keep the default for linux. I guess we have better job security options than adding `slc8` when it comes out... ;-)